src/index.js

```diff
@@ -6,15 +6,15 @@ export default class {
     isABetterThanBFunction = undefined,
     population = [],
     populationSize = 100,
-    chanceOfMutation = 50){
```

Comment:
Please fix the introduced syntax error due to merged arguments list and function body.